default/server.conf

```diff
@@ -0,0 +1 @@
+python.version = python3
```

Comment:
Don't want this to apply to the whole system so it is better at the modular input level.

Reply:
Hi, I agree it's better to set it at modular input level, but I still get the error mentioned in #19 in Splunk 7.3 about an invalid key stanza. I recommend you to verify and test the compatibility again.

Moreover, according to Splunk docs, to apply this setting to the whole system you'll have to add that line in `$SPLUNK_HOME/etc/system/local`. and not in `$SPLUNK_HOME/etc/apps/TA-mailclient/default/`.

Reply:
Can you try to do a fresh pull? Or download the last version on Splunkbase?
The current dev and master HEAD commits both passed the tests for 7.3.x and 7.2.x.

That's not how it works. If you add it to any server.conf, it will get applied to the whole system. It doesn't matter if it is in etc/system/ or etc/apps/. But you can choose to add this to your own config if you want this.

Reply:
I also tested this without adding the additional_folders = subfolder, and it seems to work.
https://travis-ci.com/github/seunomosowon/TA-mailclient/builds/211159843

Test 231 is without the additional_folders set. Test 230 as the additional_folders set. I added the else statement yesterday. You can close the PR once you test and confirm it works for you.

Reply:
Just pulled the latest version (1.5.2) from Splunkbase. This is what Splunk 7.3 logs at restart:
```bash
$ sudo /opt/splunk/bin/splunk restart
Stopping splunkd...
Shutting down.  Please wait, as this may take a few minutes.
..                                                         [  OK  ]
Stopping splunk helpers...
                                                           [  OK  ]
Done.

Splunk> Now with more code!

Checking prerequisites...
	Checking http port [8000]: open
	Checking mgmt port [8089]: open
	Checking appserver port [127.0.0.1:8065]: open
	Checking kvstore port [8191]: open
	Checking configuration... Done.
	Checking critical directories...	Done
	Checking indexes...
		Validated: _audit _internal _introspection _telemetry _thefishbucket history mail main summary
	Done
	Checking filesystem compatibility...  Done
	Checking conf files for problems...
		Invalid key in stanza [mail] in /opt/splunk/etc/apps/TA-mailclient/default/inputs.conf, line 2: python.version  (value:  python3).
		Your indexes and inputs configurations are not internally consistent. For more information, run 'splunk btool check --debug'
	Done
	Checking default conf files for edits...
	Validating installed files against hashes from '/opt/splunk/splunk-7.3.0-657388c7a488-linux-2.6-x86_64-manifest'
	All installed files intact.
	Done
All preliminary checks passed.

Starting splunk server daemon (splunkd)...
Done
                                                           [  OK  ]

Waiting for web server at http://127.0.0.1:8000 to be available... Done


If you get stuck, we're here to help.
Look for answers here: http://docs.splunk.com

The Splunk web interface is at http://so1:8000
```

As you can see, there's still this message `Invalid key in stanza [mail] in /opt/splunk/etc/apps/TA-mailclient/default/inputs.conf, line 2: python.version  (value:  python3).`
You can verify this error in Splunk with this SPL `index=_internal sourcetype=splunkd_stdout python.version`.

`additional_folders` parameter seems to be working now. Thanks for the change.